apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns.php

```diff
@@ -82,75 +107,50 @@ public function register_patterns() {
 	 *
 	 * @return array
 	 */
-	public function get_patterns() {
-		$patterns_dir = __DIR__ . '/patterns/';
-		$patterns     = array();
-
-		if ( ! is_dir( $patterns_dir ) ) {
-			return $patterns;
-		}
-
-		$directory_handle = opendir( $patterns_dir );
-		if ( ! $directory_handle ) {
-			return $patterns;
-		}
-
-		// Get all the pattern files.
-		$pattern       = readdir( $directory_handle );
-		$pattern_files = array();
-		while ( false !== $pattern ) {
-			// Only allow files ending in .json or .php.
-			if ( substr( $pattern, -5 ) === '.json' || substr( $pattern, -4 ) === '.php' ) {
-				$pattern_files[] = $pattern;
+	private function get_patterns() {
+		$block_patterns = get_transient( $this->patterns_cache_key );
+
+		// Load fresh data if we don't have any patterns.
+		if ( false === $block_patterns || ( defined( 'WP_DISABLE_PATTERN_CACHE' ) && WP_DISABLE_PATTERN_CACHE ) ) {
+			$request_url = add_query_arg(
+				array(
+					'language'   => $this->get_iso_639_locale(),
```

Comment:
Looks like `phpcs` is complaining that the `=>` in this line and the following one aren't lined up.

Reply:
Fixed :+1:

Reply:
Thanks! I'll do some more smoke testing after lunch (in a couple of hours 😃)